Attendees/Brokers/Storages/IStorageBroker.Attendees.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Attendees.Models.Foundations.Attendees;
+
+namespace Attendees.Brokers.Storages
+{
+    public partial interface IStorageBroker
+    {
+        ValueTask<Attendee> InsertAttendeeAsync(Attendee attendee);
+        IQueryable<Attendee> SelectAllAttendees();
```

Comment:
Method `SelectAllAttendees` violates async broker conventions: return type should be `ValueTask` or `ValueTask<T>`; method name should end with `Async`.